plugin/src/main/java/com/denizenscript/denizen/events/entity/EntityTeleportScriptEvent.java

```diff
@@ -17,6 +17,15 @@
 
 public class EntityTeleportScriptEvent extends BukkitScriptEvent implements Listener {
 
+    // <--[language]
+    // @name Teleport Cause
+    // @group Useful Lists
+    // @description
+    // NOTE: Teleport causes only work for player entities.

```

Comment:
Check line 177 - `"ENTITY_PORTAL" : "ENTITY_TELEPORT"` are defined special custom Denizen-specific causes (so that there's *something* when it's not a player, albeit not really detailed beyond just the two options), so that should be included in this doc

Reply:
Included this now, tried my best to keep it from being too convoluted